group_project_v2/mixins.py

```diff
@@ -316,4 +322,4 @@ class CommonMixinCollection(
     StudioEditableXBlockMixin, StudioContainerXBlockMixin,
     WorkgroupAwareXBlockMixin, TemplateManagerMixin
 ):
-    pass
+    block_settings_key = 'group_project_v2'
```

Comment:
@e-kolpakov Depending on how this is used, it might make sense to upcase `block_settings_key` (i.e., if it's used as a constant?